### ISSUE TITLE
PCB Design Files

### ISSUE THREAD

**Issue by KronosDesign** [radensb/eZ_Navi-Delete#1]
Hi radensb,

Sadly the370z forums have a registration bug atm so I can only message you here. Incredibly interested in this project but it seems like the files here are outdated compared to what was shown in the forum thread. I'm sure you're busy testing or simply have other priorities to take care of, completely understand. When you do get around to it, will you be committing the PCB design files? I'd love to open this up in Altium and mess around with the schematic. I'm eager to put this board together and do some initial testing. I have decent knowledge in C++ also, so I'd love to help out there as well.

**Comment by radensb** [comment]
@KronosDesign Thanks for the interest. Yes - life has been hectic and I am trying to get things together for my first round of assemblies. I made some progress today with the current design and getting stuff updated here. I have updated the gerbers and BOM for the current 1.0a revision of the board. I have also added the schematic to the PCB directory. I will eventually upload the design files (KiCAD), but want to hold of until I get a tagged version to fab before the design can potentially diverge.

I have another branch called 'buildConfig' that contains updated C code that I am testing. It allows for customizing the FW build parameters from a header file and uses the official STM32 HAL libs. It will soon be merged into the main branch before I start tackling the support of the various configurations that the HW was designed to support.
